101-linq-samples/docs/aggregates-3.md

```diff
@@ -2,7 +2,7 @@
 
 *Aggregator* methods return a single value calculated from all the elements in a sequence. the aggregator methods include `Count`, `Sum`, `Min`, `Max`, `Average`, and `Aggregate`.
 
-## Find the maximun of a sequence of elements
+## Find the maximum of a sequence of elements
```

Comment:
Hey Michael

This PR is too simple.
Could you make the code more complex to accommodate future changes that we don't yet know?

Reply:
@sutil-dyego 

Haha, thanks for the helpful comment. Since I'm totally clueless here, feel free to propose potential changes which allow us to be prepared for future changes <3 :D ...